### ISSUE TITLE
Bug on SiteConfig when having more than 1 tab open with different locales

### ISSUE THREAD

**Issue by chromos33** [tractorcow-farm/silverstripe-fluent#554]
Don't know if this is fixable, but I encountered a ?bug?.
When I open 2 tabs in the same browser both on the SiteConfig and I make changes to the SiteConfig with Locale X in the one tab and save and afterwards change to the tab with Locale Y and save suddenly all translated Settings/Siteconfig of Locale X are written to the Settings/SiteConfig of Locale Y.
Less of a problem for me now that I know but Customers might make that mistake.

**Comment by robbieaverill** [comment]
Hi @chromos33, this is the same problem we have with subsites (https://github.com/silverstripe/silverstripe-subsites/issues/314) in that the locale/subsite is stored in PHP sessions, so multiple tabs with different contexts interfere with eachother. There's probably a discussion thread there for how we may be able to resolve this issue.

**Comment by tractorcow** [comment]
Yeah it's probably something we can't solve in fluent easily; There are ways to address it though (e.g. hidden fields) so we could leave this issue open in case someone wants to attempt it.

**Comment by Zauberfisch** [comment]
still an issue in silverstripe 5

And critically, I am also getting this problem with Pages, not just SiteConfig
